variables.tf

```diff
@@ -39,6 +39,12 @@ variable "server_memory" {
   default     = 1024
 }
 
+variable "server_volume_size" {
+  description = "The size of the volume in bytes attached to the server"
+  type        = string
+  default     = "10737418240"
```

Comment:
Why not a `number`?

Reply:
I asked myself the same question but in the end I stupidly copied the existing format

Reply:
@mcanevet I just tested/updated this PR with numbers, it seems to work as expected